examples/docstring_reference.py

```diff
@@ -6,3 +6,10 @@ def my_great_function():
     http://doi.org/10.5281/zenodo.1185316
     """
     pass
+
+
+def my_great_function_2():
```

Comment:
Can we have different files for different features?

Reply:
Sure, how about `docstring_doi_reference.py` and `docstring_sphinx_reference.py`?